ricecooker/classes/files.py

```diff
@@ -356,14 +358,32 @@ class AudioFile(DownloadFile):
     def get_preset(self):
         return self.preset or format_presets.AUDIO
 
+
 class DocumentFile(DownloadFile):
     default_ext = file_formats.PDF
-    allowed_formats = [file_formats.PDF]
+    allowed_formats = [file_formats.PDF, file_formats.EPUB]
     is_primary = True
 
     def get_preset(self):
         return self.preset or format_presets.DOCUMENT
 
+
+class EPubFile(DownloadFile):
```

Comment:
When would one use the DocumentFile vs the EPubFile? Looks like epub is accepted under the document file, but wondering if it would make sense to just keep these separate entirely to avoid any confusion